### PR TITLE
Added xunitSettings.props to all test assemblies

### LINKED ISSUE #4076: Try to reduce number of failing time-sensitive tests with running all tests in sequence
Need to reference `xunitRunner.props` from all test specs projects and see if this will help.

### PR DESCRIPTION
**Pull request by IgorFedchenko**
Close #4076

Let's just see how this will help with running racy tests in this PR

### CODE REVIEW

File: src/xunitSettings.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <!-- use the xunit JSON settings file -->
   <ItemGroup>
-    <Content Include="..\..\xunit.runner.json" Link="xunit.runner.json">
+    <Content Include="$(SolutionDir)xunit.runner.json" Link="xunit.runner.json">
```
Comment (Aaronontheweb) [inline src/xunitSettings.props]: Good change.

File: src/core/Akka.Remote.Tests.MultiNode/Akka.Remote.Tests.MultiNode.csproj [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\common.props" />
+  <Import Project="..\..\xunitSettings.props" />
```
Comment (Aaronontheweb) [inline src/core/Akka.Remote.Tests.MultiNode/Akka.Remote.Tests.MultiNode.csproj]: Won't have any impact on the MNTR - those tests are already single threaded.

Comment (IgorFedchenko) [reply src/core/Akka.Remote.Tests.MultiNode/Akka.Remote.Tests.MultiNode.csproj]: Will remove this entries once CI will finish current run. It is going well so far

Comment (IgorFedchenko) [reply src/core/Akka.Remote.Tests.MultiNode/Akka.Remote.Tests.MultiNode.csproj]: Will remove this entries once CI will finish current run. It is going well so far

**Comment by Aaronontheweb** [comment]
NodeChurn spec failed, but we already know that one has issues: https://github.com/akkadotnet/akka.net/pull/4150

**Comment by Aaronontheweb** [comment]
Overall I agree with you @IgorFedchenko - these results look promising. Kicked off another build with the latest `dev` changes.

**Comment by IgorFedchenko** [comment]
Another thing we could make here is running PR Validation steps sequentially instead of running in parallel. But on the other hand, tasks should be running in separate containers (they even use different docker images) - so they should be executed independently anyway.

**Comment by IgorFedchenko** [comment]
So this run has NodeChurn  and 2 more specs that are known to be racy. Looks like any other CI run on other PRs at this time.

All right, seems like we may need this PR merged, but only after some nice work on #3786. After tests will be less racy (if not fixed at all) they still may fail when running with big concurrency, but without xunit collections parallelization may behave well.

@Aaronontheweb What do you think?

**Comment by IgorFedchenko** [comment]
Actually, let's make some more runs to verify that there is any more or less stable improvement.

**Comment by IgorFedchenko** [comment]
Ok, agree with you. Let me just remove those `xunitSettings.props` from MNTR test projects

**Pull request merged**